msbuild/Xamarin.MacDev.Tasks/Tasks/ParseBundlerArgumentsTaskBase.cs

```diff
@@ -165,6 +168,9 @@ public override bool Execute ()
 						item.SetMetadata ("Value", value.Substring (colon + 1));
 						envVariables.Add (item);
 						break;
+					case "skip-marking-nsobjects-in-user-assemblies":
+						SkipMarkingNSObjectsInUserAssemblies = ParseBool (value) ? "false" : "true";
```

Comment:
This looks weird. If the extra arg `skip-marking-nsobjects-in-user-assemblies` is **true**, we set `SkipMarkingNSObjectsInUserAssemblies` to **false**?

Reply:
@emaf Good catch! I've now fixed this.